src/dymoprint/gui.py

```diff
@@ -56,7 +57,7 @@ def __init__(self):
 
     def init_elements(self):
         self.setWindowTitle("DymoPrint GUI")
-        self.setWindowIcon(QIcon(str(ICON_DIR / "gui_icon.png")))
+        self.setWindowIcon(QIcon(str(ICON_DIR.joinpath("gui_icon.png"))))
```

Comment:
Actually `ICON_DIR / "gui_icon.png"` is the modern `pathlib` style.

Reply:
Thanks for this. You are right, and I actually overlooked this. 
I confused it to be a change in the style of chaging `ICONDIR + '/gui_icon.png'`.